app/Tasks.php

```diff
@@ -12,7 +12,28 @@ class Tasks extends Model
 
     public function deleteTask($id)
     {
-        return self::where('id', $id) -> delete();
+        $message = "";
+        $task = $this->Find($id);
+
```

Comment:
use findORfail() instead of find() that way error is handled by laravel instead of u having a null check